tests/dashbio_demos/dash-pileup/assets/dash-pileup.css

```diff
@@ -0,0 +1,100 @@
+#pileup-body ::-webkit-scrollbar-thumb {
+    background-color: #34AD99 !important;
+}
+
+/* pileup track */
+.pileup-root > .pileup, .pileup-root > .genotypes {
+  flex: 1;  /* stretch to fill remaining space */
+  flex-basis: auto;
+  min-height: 200px !important; /* prevent tracks from becoming too small */
+}
+
```

Comment:
```suggestion

.pileup-root {
   padding: 20px;
}

```

We'll want to add a little bit of padding to the root container, so it doesn't cut off onto the volcano plot or other parts of the app:

![image](https://user-images.githubusercontent.com/30986043/111838260-78ba1180-88cf-11eb-881c-9e4436e3666b.png)